### ISSUE TITLE
comic vine id from filename?

### ISSUE THREAD

**Issue by fieldhouse** [bpepple/thwip#8]
If this would be a big change, feel free to roundfile.  

I have comic archives that shouldn't be changed for data consistency purposes (checksums change, etc) but it would be trivial to add the comicvine id info to the filename.  

Would it make sense to support checking the filename for something like  `[CVDB154258]`?  I guess it could be `[4000-154258]` as well but that seems redundant and non-specific.

**Comment by bpepple** [comment]
So, your comic archives don't have any metadata in them? Do you have some custom script that is rewriting the filename with data from Comic Vine, or is comictagger (or similar utility) providing that functionality these day?

It wouldn't be terribly hard to get the Comic Vine id from the file name, tho it does seem to be quite a corner-case scenario to implement.

**Comment by fieldhouse** [comment]
good question.  I've been using ComicRack with the ComicVine scraper plugin and all the data is stored in a database.  I haven't checked whether comictagger supports it but I kind of doubt it does.  Most tools treat cbx files like mp3's.

Totally understand it being an uncommon case.  I mostly threw it out there in case there was other interest.  I may need to bite the bullet one of these days and inject ComicInfo into my whole library.

**Comment by drose-github** [comment]
I actually did a similar thing with adding cvdb to the filename.  I ended up adding the volume ID to the folder name for each series.
The reason was because Marvel was just having a jolly good time releasing the same series name within a single year (see Hawkeye, Squirrel Girl) and it was very difficult to keep those issues organized properly.

However I keep all the metadata as well.  It just makes it easier for scraping.
